### PR TITLE
Reorganize pipelines

### LINKED ISSUE #63: Reorganize pipelines
Reorganize pipelines from types to be named pipelines.

**Original organization**
```
draco/pipelines
  ├── classes
      ├── normalize_dfs_xgb_classifier.json
      ├── unstack_dfs_xgb_classifier.json
      ├── unstack_double_lstm_timeseries_classifier.json
      ├── unstack_lstm_timeseries_classifier.json
      └── unstack_normalize_dfs_xgb_classifier.json
  ├── probability
      ├── normalize_dfs_xgb_classifier.json
      ├── unstack_dfs_xgb_classifier.json
      ├── unstack_double_lstm_timeseries_classifier.json
      ├── unstack_lstm_timeseries_classifier.json
      └── unstack_normalize_dfs_xgb_classifier.json
  ├── unstacked
      ├── unstacked_dfs_xgb_classifier.json
      ├── unstacked_double_lstm_timeseries_classifier.json
      ├── unstacked_lstm_timeseries_classifier.json
      └── unstacked_normalize_dfs_xgb_classifier.json
```

**Proposed organization**
```
draco/pipelines
  ├── dfs_xgb
      ├── dfs_xgb.json
      ├── dfs_xgb_with_unstack.json
      ├── dfs_xgb_with_double_normalization.json
      └── dfs_xgb_with_unstack_normalization.json
  ├── lstm
      ├── lstm.json
      └── lstm_with_unstack.json
  ├── double_lstm
      ├── double_lstm.json
      └── double_lstm_with_unstack.json
```

In addition, for each pipeline, create a documentation page describing the pipeline.

### PR DESCRIPTION
**Pull request by sarahmish**
Resolve #63

### CODE REVIEW

**Comment by dyuliu** [comment]
Remove py3.6 in the readme.

File: draco/pipeline.py [hunk 1]
```diff
@@ -66,25 +66,33 @@ def get_pipelines(pattern='', path=False, pipeline_type='classes'):
         path (bool):
             Whether to return a dictionary containing the pipeline
             paths instead of only a list with the names.
-        pipeline_type (str):
-            The pipeline category to filter by (`classes`, `probability` and `unstacked`).
-            Defaults to `classes`.
+        pipeline_type (str or list[str]):
+            The pipeline category to filter. Defaults to `None`.
```
Comment (dyuliu) [inline draco/pipeline.py:70]: what are the possible types? (`classes`, `probability` and `unstacked`)?

File: draco/pipelines/double_lstm/double_lstm.json [hunk 2]
```diff
@@ -34,17 +34,13 @@
             "key": "timestamp"
         },
         "mlprimitives.custom.timeseries_preprocessing.cutoff_window_sequences#1": {
-            "window_size": 72,
+            "window_size": 24,
```
Comment (dyuliu) [inline draco/pipelines/double_lstm/double_lstm.json:37]: Is this value selected based on a certain strategy?

Comment (sarahmish) [reply draco/pipelines/double_lstm/double_lstm.json:37]: It's only a template, and the data we have is resampled by 1 hour, so I made the window size 24 to express 1 day. The reason I made this change is to have faster testing.

File: draco/pipeline.py (hunk 1 above)
Comment (sarahmish) [reply draco/pipeline.py:70]: It used to mean the pipeline type which is:
* `classes` was to predict the label.
* `probability` was to predict the probability of a label.
* `unstacked` was to assume that the data is already unstacked. 

This definition is gone, and I made `pipeline_type` refer to the type of the model. So for example you can filter by `dfs_xgb`, `lstm`, `double_lstm`, etc.

**Comment by sarahmish** [comment]
> Remove py3.6 in the readme.

We still keep py3.6, I re-added it to the tests and the setup of Draco. I think we need more time before fully removing it from the support.

**Review by dyuliu — APPROVED** [review]

**Review by lcwongme — APPROVED** [review]

**Pull request merged**